crates/matrix-sdk-common/src/linked_chunk/mod.rs

```diff
@@ -1440,6 +1409,17 @@ mod tests {
         LinkedChunk, Position,
     };
 
+    trait LinkedChunkLenExt {
+        /// Returns the number of items of the linked chunk.
+        fn len(&self) -> usize;
+    }
+
+    impl<const CAP: usize, Item, Gap> LinkedChunkLenExt for LinkedChunk<CAP, Item, Gap> {
+        fn len(&self) -> usize {
+            self.items().count()
+        }
+    }
+
```

Comment:
Why a trait and not a normal function in the `impl` block of `LinkedChunk` that is guarded by a `#[cfg(test)]` attribute?

Reply:
Could even be a public function if we wanted to, to maintain backwards compat.

Reply:
Wouldn't mind that either, it's just that the trait feels heavy handed for what we're achieving here.

Reply:
The good part is that it makes it a non-breaking change :sunglasses: